### ISSUE TITLE
Breaking website functionality

### ISSUE THREAD

**Issue by lucaben** [codebicycle/videospeed#90]
I've recently had lots of problems with websites who would not work properly, and finally tested all add-ins and found out the culprit is this extension. It was very disappointing to find that out, because it's proved to be tremendously useful so far, but this has caused lots of frustration and time loss, so I will have to disable it.
 
Examples:
- https://westernrise.com/: checkout from this website completely blocked (same on other websites using shopify)
- https://www.vykingship.com/, after login, form controls not appearing and buttons not working

**Comment by kolys22** [comment]
Yes, This extension breaks or blocking some websites! I give some examples too:

https://www.flightradar24.com
If i click on any aircraft, more details should appear on the left side but nothing is happening.

https://www.usdebtclock.org/world-debt-clock.html
On this website numbers and any data are not visible sometimes. 

Everything start working properly when I disable or delete this extension.

**Comment by codebicycle** [comment]
Thanks for taking the time to investigate.
Version 0.6.1 was released and should address this issue.
